Resources/Prototypes/Entities/Objects/Specific/Mech/mechs.yml

```diff
@@ -99,6 +102,68 @@
   parent: MechRipley
   suffix: Battery
   components:
+  - type: ContainerFill
+    containers:
+    mech-battery-slot:
+    - PowerCellHigh
+
+  - type: Mech
```

Comment:
dont need Mech component at all for battery variant

Reply:
should be sorted 👍